ctrd/container.go

```diff
@@ -563,7 +563,8 @@ func (c *Client) KillContainer(ctx context.Context, id string, sig int) (*Messag
 	}
 
 	// if container task has been stopped, we should wait container and task to be totally deleted
-	time.Sleep(10 * time.Millisecond)
+	// FIXME: This wait process consumes too much time
+	time.Sleep(50 * time.Millisecond)
```

Comment:
it is a trick !!!!  we should use `ProbeContainer` here, WDYT?

Reply:
Do you mean that we should use `ProbeContainer` to judge whether this container is killed or not? It is indeed better than this. But if the container is not killed by client signal, the `ProbeContainer` process will wait until the `TimeOut Error` occurs, so we should set `time.Duration` to a proper value and this value can't be too long, because we shouldn't make client wait too long. A value in my mind is `100ms`, WDYT?